content/docs/forwarding-refs.md

```diff
@@ -37,6 +37,12 @@ Here is a step-by-step explanation of what happens in the above example:
 >
 >Ref forwarding is not limited to DOM components. You can forward refs to class component instances, too.
 
+## Warning for third-party library maintainers
+
+**If you start using `forwardRef` in a third-party library, you should release it as a breaking change.**.  This is because the value returned by `forwardRef` is of a different type than React components have historically been (a `class` extending `React.Component` or a `function`) and many HoC libraries (e.g. `react-redux`' `connect()`) aren't currently compatible with such components.
```

Comment:
I would just leave it at "has an observably different behavior (including the component type and what the ref gets assigned to), and can break apps and other libraries that depend on the old behavior." I don't think there's a need for jargon like "HoC" or mentioning specific libraries.

Reply:
okay, sounds good